built/cinematic.js

```diff
@@ -344,6 +345,46 @@ var Cinematic = /** @class */ (function () {
                 this.title = me.options.translations.hideSubtitles;
             }
         });
+        document.addEventListener('keyup', function (event) {
+            var key = event.key;
+            switch (key) {
+                // Spacebar allows to pause/resume the video
+                case ' ':
+                    if (_this._video.paused) {
+                        _this._video.play();
+                    }
+                    else {
+                        _this._video.pause();
+                    }
+                    break;
+                // Left Arrow skips 10 seconds into the past
+                case 'ArrowLeft':
+                    _this._video.currentTime -= 10;
+                    break;
+                // Right Arrow skips 10 seconds into the future
+                case 'ArrowRight':
+                    _this._video.currentTime += 10;
+                    break;
+                // Down Arrow decreases the volume by 5%
+                case 'ArrowDown':
+                    if (_this._video.volume > 0) {
+                        var currentVolume = Math.round((_this._video.volume + Number.EPSILON) * 100);
+                        _this.volume = (currentVolume - 5) / 100;
+                        _this._video.volume = _this.volume;
+                        _this._volumeSlider.value = _this.volume.toString();
+                    }
```

Comment:
If we are at a volume of 3%, which could be possible if we selected the volume with a cursor later on, we would end up at -2%. We should either check if we are below 5% and simply set it to 0 or set it to 0 if we end up in the negatives.

Same goes for volume up

Reply:
There should never be the possibility to reach a volume setting of 3% as the keyboard shortcuts change it by 5% increments and the slider is stepped so it also only changes it only by 5% increments. Or am i missing a way to alter it?